rfcs/convergence/authoring-stories.md

```diff
@@ -433,6 +434,49 @@ export const Default = (props: PopoverProps) => (
 Public stories should only contain code, which is useful for users to see after clicking on “Show code” in documentation.
 Extra markup (e.g., container with CSS styles) can be added via [Decorators](https://storybook.js.org/docs/react/writing-stories/decorators).
 
+### 10. Internal stories for testing
+
+As mentioned in for E2E testing, we should ensure maximum coverage for all publicly viewable stories by our consumers.
+For more complex scenarios that need to be tested we should make sure that stories exist for E2E tests but should not be
+easily accessible publicly.
+
+Storybook has proposed a feature for this in [storybookjs/storybook#9209](https://github.com/storybookjs/storybook/issues/9209)
+which will configure stories to exist in deeplink URL format, but do not appear in the nav tree or the docs page. As stated in the issue,
+we can workaround before the release of this feature by modifying `manager-head.html` and set `display:none`for all
+stories with a specific DOM `id` attribute. Storybook uses the `id` attribute for each link in the nav tree, and sets
+the value to the story id.
+
+We propose to use an extra filename extension and naming convention for internal stories:
+
+```ts
+// TabstopsInternal.stories.tsx
```

Comment:
**chore:** This should be `MenuTabstopsInternal.stories.tsx` according to 8.2

Reply:
Done in [This commit](https://github.com/microsoft/fluentui/pull/19333/commits/f8510d8a0bab382b81989a436bcc602e8958a0e1)